Network Class/Network.cpp

```diff
@@ -178,6 +176,60 @@ bool backProp(int index)
 	return correct;
 }
 
+// checks if Network output matches expected
+// @param - takes a matrix of expected vals (could be changed to vector though or process both if need be)
+// returns <0 if correct, >0 if incorrect and a 0 if ambiguous
+bool Network::compareOutput(const Matrix& expectedValues) 
+{
+	int lastInd = numLayers - 1;
+	int lastSize = layerSizes[lastInd];
+
+	// indices at which there is the biggest activation or a 1 for the netowrk or expected vals respectively
+	int biggestI = -1; int expectedI = -1;
+
+	// biggest starts at 0 (not MIN) because any number < 0 might as well be 0 for these purposes
```

Comment:
The biggest has to start with the -1 or min negative value possible because when we change the activation function we might get values from -1 to 1 (eg: tanh())

Reply:
I am sorry, I didn't read the complete function. But yeah, that makes sense now!